renku/api/storage.py

```diff
@@ -61,88 +61,106 @@ def init_external_storage(self, force=False):
     @property
     def external_storage_installed(self):
         """Check that Large File Storage is installed."""
-        return HAS_LFS and self.repo.config_reader(
-        ).has_section('filter "lfs"')
+        return HAS_LFS
 
     def track_paths_in_storage(self, *paths):
         """Track paths in the external storage."""
-        if self.use_external_storage and self.external_storage_installed:
-            track_paths = []
-            attrs = self.find_attr(*paths)
-
-            for path in paths:
-                # Do not add files with filter=lfs in .gitattributes
-                if attrs.get(path, {}).get('filter') == 'lfs':
-                    continue
-
-                path = Path(path)
-                if path.is_dir():
-                    track_paths.append(str(path / '**'))
-                elif path.suffix != '.ipynb':
-                    # TODO create configurable filter and follow .gitattributes
-                    track_paths.append(str(path))
-
-            call(
-                self._CMD_STORAGE_TRACK + track_paths,
-                stdout=PIPE,
-                stderr=STDOUT,
-                cwd=str(self.path),
-            )
-        elif self.use_external_storage:
+        if not self._use_lfs():
+            return
```

Comment:
Let's make use_lfs method as part of the public API, cause I think we will need it in #588 and here we should raise an exception and not silently fail.